src/components/WeLiveInAWorld.tsx

```diff
@@ -0,0 +1,23 @@
+import { AccentText, BodyText, HeaderText } from 'components/Text'
+import { space } from 'classnames/tailwind'
+import Card from 'components/Card'
+import PrismOrb from 'components/PrismOrb'
+
+const cardBody = space('space-y-6')
+
+export default function () {
+  return (
+    <Card tall nospace orb>
```

Comment:
Did you check it on the Galaxy Fold screen? Won't it be any problems with elements? Especially with `ProsmOrb`.